packages/twenty-front/src/modules/apollo/hooks/useApolloFactory.ts

```diff
@@ -54,6 +59,7 @@ export const useApolloFactory = (options: Partial<Options<any>> = {}) => {
       connectToDevTools: process.env.IS_DEBUG_MODE === 'true',
       currentWorkspaceMember: currentWorkspaceMember,
       currentWorkspace: currentWorkspace,
+      appVersion: sentryConfig?.release ?? '',
```

Comment:
This feels ugly as we shouldn't rely on Sentry for this which isn't related to Sentry, I'm sure there's a cleaner way already :)

Reply:
I checked and didn't see anything ; let's just create another property in Client config!